addons/account/security/ir.model.access.csv

```diff
@@ -40,7 +40,7 @@ access_account_invoice_group_invoice,account.invoice group invoice,model_account
 access_res_currency_account_manager,res.currency account manager,base.model_res_currency,group_account_manager,1,1,1,1
 access_res_currency_rate_account_manager,res.currency.rate account manager,base.model_res_currency_rate,group_account_manager,1,1,1,1
 access_account_invoice_user,account.invoice user,model_account_invoice,base.group_user,1,0,0,0
-access_account_invoice_user,account.invoice.line user,model_account_invoice_line,base.group_user,1,0,0,0
+access_account_invoice_line_user,account.invoice.line user,model_account_invoice_line,base.group_user,1,0,0,0
```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/account/security/ir.model.access.csv#L42